scripts/templates/generate_coverage_report

```diff
@@ -112,6 +112,10 @@ if [[ "$1" == "--codecov" ]]; then
       exit 1
    fi
 
+   # Remove other files to be sure CodeCov uses only the coverage.info file
```

Comment:
Unnecessary change: these lines were blow the TOKEN check. Please, don't move them around or the diff gets tricker than necessary to review.